toolbox/certbot/README.md

```diff
@@ -0,0 +1,15 @@
+# Airflow SSL scripts
+
+For SSL termination management of the resources in this Airflow stack, we use [certbot](https://certbot.eff.org/).
+
+## Request certificate
+
+The request script has already been run when Airflow was set up on atd-data03. The initial certificate expires three months from its request. The request command is identical to the certbot command used in the renew script.
+
+## Renew certificate
+
+The renew script is set up as a cron job with a monthly frequency with output logged to `/var/log/airflow_cert_renewal.log`. The `renew.sh` script renews all of the domains used by the stack using the `renew_domain_with_certbot.sh` script that takes one argument like:
```

Comment:
One thing that just occurred to me that you might want add here is that the renewal script must be run as root. Certbot's output in `/etc` is owned by root and set so that only root can read it because it contains private keys and other sensitive information. 

You could maybe even put something like this at the top of the script to remind users if they forget:

```
# Check if the script is running as root
if [ "$EUID" -ne 0 ]
  then echo "Please run as root"
  exit
fi
```

Reply:
I ended up adding this to the readme. Thanks for the reminder! 🙏